packages/paste-libraries/react-textarea-autosize/src/__tests__/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import TextareaAutosize from '../index';
+
+describe('<TextareaAutosize />', () => {
+  it('renders ok', () => {
+    const {asFragment} = render(<TextareaAutosize />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  // it('renders with initial height passed in style prop', () => {
+  //   const props = {
+  //     style: {
+  //       height: 55,
+  //     },
+  //   };
+  //   const {asFragment} = render(<TextareaAutosize {...props} />);
+  //   expect(asFragment()).toMatchSnapshot();
+  // });
+});
```

Comment:
question: is there a plan to add tests to cover the functionality of this package? There are some hooks and utils etc that could do with some tests

Reply:
I agree that it could! These are the only tests the library had and I just copied them over.

Is this enough of a baseline for now or should I add them in this PR?

Reply:
I think a good starting point would be at least the functions like `getSizingData`, `getHeight`, `calculateNodeHeight`, `forceHiddenStyles`, `useWindowResizeListener` etc

Just felt a bit weird pointing a bunch of "missing unit test" tickets whilst adding code without unit tests 😬

Reply:
Lol fair point - I'll add them today!